src/Makefile

```diff
@@ -140,7 +140,7 @@ dist:
 
 distclean: clean
 	rm -f kak kak$(suffix)
-	find ../doc -type f \( -name \*\\.gz -o -name \*\\.1 \) -exec rm -f '{}' +
+	find . ../doc -type f \( -name \*\\.gz -o -name \*\\.1 \) -exec rm -f '{}' +
```

Comment:
This changes the meaning of the command, it will search from the current directory in addition to the doc directory which is not the intention here.

Reply:
You're right, thanks. I've pushed the update to undo this.

Reply:
👍